src/explanation/models.py

```diff
@@ -14,13 +14,17 @@ class ExplanationTypes(Enum):
     LIME = 'lime'
     TEMPORAL_STABILITY = 'temporal_stability'
     ANCHOR = 'anchor'
+    ICE = 'ICE'
```

Comment:
why this is upper case whereas the other three aren't? please make it lower case